ecl/hql/hqlexpr.hpp

```diff
@@ -836,6 +836,7 @@ interface IFileContents : public IInterface
     virtual ISourcePath * querySourcePath() = 0;
     virtual const char * getText() = 0;
     virtual size32_t length() = 0;
+    virtual bool isImplicitlySigned() = 0;
```

Comment:
minor: could have been a const function.  But then so could some of the other functions in this interface...